### PR TITLE
Home work 06

### PR DESCRIPTION
**Pull request by pernarovskyi**

### CODE REVIEW

File: controllers/auth-controller.js [hunk 1]
```diff
@@ -48,44 +63,86 @@ const signIn = async (req, res) => {
...
+const verifyToken = async (req, res) => {
+  const { verificationToken } = req.params;
+  
+  const user = await User.findOne({verificationToken});
+  
+  if (!user) {
+    throw HttpError(404, "User not found.");
+  }
+  
+  await User.findByIdAndUpdate(user._id, {verificationToken: "", verify: true});
```
Comment (ceREAL133) [inline controllers/auth-controller.js]: verificationToken should be null

**Review by ceREAL133 — CHANGES_REQUESTED** [review]

File: controllers/auth-controller.js (hunk 1 above)
Comment (pernarovskyi) [reply controllers/auth-controller.js]: fixed

Comment (ceREAL133) [reply controllers/auth-controller.js]: have you tested your changes? Will it work correctly?

Comment (pernarovskyi) [reply controllers/auth-controller.js]: [Photo of test](https://postimg.cc/zHyDb2Jt)

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**